### ISSUE TITLE
Project COM getters and setters as properties

### ISSUE THREAD

**Issue by mcooley** [microsoft/CsWin32#738]
Consider APIs like Windows.UI.Accessibility.IUIAutomationElement.get_CachedAcceleratorKey. This should be exposed to .NET developers as a _property_ instead of a method. The `get_` prefix should be omitted.

I want this so I can use cswin32 to seamlessly replace interop assemblies generated by tlbimp. This change would also make cswin32 more consistent with the Rust win32 projection ([which also omits the get_ prefix](https://microsoft.github.io/windows-docs-rs/doc/windows/Win32/UI/Accessibility/struct.IUIAutomationElement.html#method.CachedAccessKey)) and more consistent with CsWinRT.

The metadata already puts the SpecialName attribute on methods which need this treatment.

**Comment by AArnott** [comment]
I think this is an excellent suggestion. And we will usually be able to do it. The exception will be when the virtual method table describes the property getter and setter, but not in neighboring rows in the table. In such a case, CSWin32 at best can emit only one of them as a property.

**Comment by AArnott** [comment]
Trivia: `IUIAutomationProxyFactoryEntry` is an example of an interface that doesn't put the accessors next to each other. As a result, there is no C# representation of the interface where both accessors can be represented on the property.
